packages/dev/inspector/src/components/actionTabs/tabs/propertyGrids/parentPropertyGridComponent.tsx

```diff
@@ -34,9 +33,8 @@ export class ParentPropertyGridComponent extends React.Component<IParentProperty
         const sortedNodes = scene
             .getNodes()
             .filter((n) => n !== node)
-            .map((n) => this._getNameForSorting(n))
-            .sort((aName, bName) => {
-                return aName.localeCompare(bName);
+            .sort((a, b) => {

```

Comment:
I'm not sure why the previous code was not ok?

Reply:
`map` was transforming it in a string array, but the code needed a Node array